l10n_fr_department_delivery/README.rst

```diff
@@ -0,0 +1,30 @@
+French Departments - Delivery
+=============================
+
+This module adds a field *French Departments* on delivery grids.
+
+Configuration
+=============
+
+To configure this module, go to *Warehouse > Configuration > Delivery > Delivery Pricelist* and, in the *Destination* tab, you will find the field *French Departments*.
+
```

Comment:
Please indicate that this moduke is incompatible with other modules whuch extend the selection of carriers, in a "known problems" section ?